common/djangoapps/enrollment/data.py

```diff
@@ -15,8 +15,11 @@
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 from student.models import (
     CourseEnrollment, NonExistentCourseError, EnrollmentClosedError,
-    CourseFullError, AlreadyEnrolledError, CourseEnrollmentAttribute
+    CourseFullError, AlreadyEnrolledError, CourseEnrollmentAttribute,
```

Comment:
We try to only add new lines, instead of editing existing lines, if at all possible for edx code;
it reduces the risk of future conflicts.

Please split this out into separate import statements:

```diff
    CourseFullError, AlreadyEnrolledError, CourseEnrollmentAttribute
)
+ from student.models import UserProfile
+ from student.models import anonymous_id_for_user
```